test/system/StringTest.ooc

```diff
@@ -11,6 +11,15 @@ use unit
 StringTest: class extends Fixture {
 	init: func {
 		super("String")
+		this add("literal", func {
+			// The & operand tries to free the literals but if the lock works then they should resist the attempt
+			result: String
+			for (i in 1 .. 100) {
+				result = "ab" & "cd"
```

Comment:
For even better effect you could declare `ab = "ab"` and `cd = "cd"` outside the loop and use them.